plugins/chat/assets/stylesheets/desktop/chat-message-actions.scss

```diff
@@ -1,5 +1,5 @@
-.chat-message-actions[data-popper-reference-hidden],
-.chat-message-actions[data-popper-escaped] {
+.chat-message-actions-container[data-popper-reference-hidden],
+.chat-message-actions-container[data-popper-escaped] {
   visibility: hidden;
   pointer-events: none;
 }
```

Comment:
This has regressed long time ago, it was supposed to hide the popper when outside of viewport. It's not fixing our issue, although fixing this makes it less likely to happen.